src/cozy/react/components/BackButton/index.jsx

```diff
@@ -0,0 +1,18 @@
+/* global cozy */
+
+import React from 'react'
+import PreviousIcon from 'cozy-ui/transpiled/react/Icons/Previous'
+
+const { BarLeft } = cozy.bar
+
+const previousIconStyle = { width: 48, padding: 16, fill: 'var(--secondaryTextColor)' }
```

Comment:
Shouldn't we set the unit for `width` and `padding`? This seems to be the react-native notation here. I didn't know this whould work in the web environment.

nit: We should prefer `em` units over `px` ones